features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -37,6 +37,7 @@ Scenario: Supplier is not eligible as they can not provide the developer role
   And I see 'You can’t apply for this opportunity because you didn’t say you could provide this specialist role when you applied to the Digital Outcomes and Specialists 2 framework.' text on the page
   And I see a 'data-reason' attribute with the value 'supplier-not-on-role'
 
+@skip-local @skip-preview
```

Comment:
Minor request but could we have the stage duplicates in the same order to avoid confusion? The tests are long and by the time I've got to the bottom I've forgotten what tag I'm on 😸  Elsewhere in the file we do `@skip-staging` then `@skip-local @skip-preview` but as long as it's consistent I'm fine either way:

```
@skip-staging
Scenario A
...
@skip-local @skip-prevew
Scenario A
...
@skip-staging
Scenario B
...
@skip-local @skip-prevew
Scenario B
...
```

Reply:
Good point, thanks for calling out my laziness